### PR TITLE
Add Features branch with changes to code base (Linters , HTML&CSS files aswell as README Description.

### PR DESCRIPTION
**Pull request by andreacostabz**
# In this pull request I've added:

1. Made a GitHub repo titled "hello-microverse"
2. Add a features branch 
3. Integrated Linters into the code base 
4. Add 3 files to code base (Index.html , styles.css,text.md)
5. Add a descriptive GitHub README page describing the project.

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## This file should be ignored by git
```
Comment (andersonlebon) [inline test.md]: - [x] Nice Job  👏  👏  👏   on following the requirements by putting the `test.md` file in the `.gitignore` file however one last thing to fix is to remove the test.md file should not appear in this repository. That is the importance of using the .gitignore all files listed inside should not appear in the repository. 
image

File: .github/workflows [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (andersonlebon) [inline .github/workflows]: - [x] Good job 👏 👏  on adding Github actions (`.github` folder). However, the path you used is not correct. kindly I would recommend you put all these configurations in a `linters.yml` file then put it inside the` workflows `folder at the end you will have a structure like this down 👇  👇 
image
For more clarifications use this [link](https://github.com/microverseinc/linters-config/tree/master/html-css)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
```
Comment (andersonlebon) [inline index.html]: ## [OPTIONAL]

- [x] Your HTML page should have a descriptive title and not be empty. I kindly suggest adding it.

File: README.md [hunk 4]
```diff
@@ -1 +1,21 @@
...
+👤 Author1
+
+GitHub: @andreacostabz
+Twitter: @andrebze
+LinkedIn: Andre Acosta
+
+Acknowledgments 
+
+Linters : Microverse
+Inspiration : Microverse
```
Comment (andersonlebon) [inline README.md]: - [x] To make the README file look more professional I would recommend following the Microverse [README template](https://github.com/microverseinc/readme-template), then customize it with your own information.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @andreacostabz 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- Good commit history :heavy_check_mark:
## Required Changes ♻️

- [x] One of the requirements says that you should add linters in your project and all of them should pass, this will help you track some basic errors in your project. check here 👇 👇  what should be shown at the bottom of this your pull request
image

- [x] The branch title should describe in a few words what was done in this pull request for example(`add the navigation bar`, `add the content of the project`...)

- [x] As your pull request description should list all the tasks that you did in this branch, I think it will be nice if you put them in a list rather than use them as a headings.

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
Comment (andreacostabz) [reply README.md]: Thank you! working on it.

File: test.md (hunk 1 above)
Comment (andreacostabz) [reply test.md]: hey Anderson! finally got everything to meet the requirements. could you have another look at my code? Thanks so much for clarifying my missing points, I really do appreciate your feedback.

**Comment by andreacostabz** [comment]
Hey Anderson! I'm all set and ready for a review, I tried to request one from my dashboard but didn't see an option to request a second review. please let me know your thoughts on the changes I made.  I was able to pass all checks and hide my test.md file aswell. It took me a while to figure it out but got it done nonetheless. 

Hope to hear from you soon.

File: .github/workflows (hunk 2 above)
Comment (andreacostabz) [reply .github/workflows]: changed directory as you have shown and followed the correct path.

**Comment by andersonlebon** [comment]
Hello @andreacostabz

**Comment by andreacostabz** [comment]
hey @andersonlebon! hope youre doing well. were the relevant issues fixed and can i go ahead and merge?

**Comment by andersonlebon** [comment]
Hey @andreacostabz  kindly your project needs to be approved before merging! You need to resubmit it for a second review

**Comment by andreacostabz** [comment]
@andersonlebon  thank you for getting back to me. could you give me guidance as to how to ask for another review?? i cant see an option to ask for another review. Help on that would be appreciated.

**Comment by andersonlebon** [comment]
Yes the same project you need to resubmit your pull request link

**Comment by andreacostabz** [comment]
https://github.com/andreacostabz/hello-microverse/pulls
@andersonlebon do i just send you the link like this? I don't have a resubmit option on my microverse dashboard.

**Comment by andreacostabz** [comment]
Just a bit confused on the resubmit process

**Comment by andersonlebon** [comment]
Click on action then click on request review

**Comment by andreacostabz** [comment]
cant request a new review
it wont give me that option. it only says changes required and submit but when i click submit it take me to a page to rate you.

**Comment by andreacostabz** [comment]
@andersonlebon I was able to resubmit! thanks for your help!

File: README.md [hunk 5]
```diff
@@ -1 +1,35 @@
...
+</ul>
+
+
+<h2><Authors/h2>
+<p>👤 Author1</p>
+
+<ul>
+   <li>GitHub: @andreacostabz</li> 
+    <li>Twitter: @andrebze</li>
+   <li>LinkedIn: Andre Acosta</li>
```
Comment (Deepakdanger) [inline README.md:22]: [OPTIONAL]
- A Good way of GitHub, Twitter & Linkedin is by providing links beside it. for example 👇 
`GitHub: [@githubhandle](https://github.com/Username)`

**Review by Deepakdanger — APPROVED** [review]
Hi @andreacostabz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- Check the comment below.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 5 above)
Comment (andreacostabz) [reply README.md:22]: awesome!! Thank you!

**Pull request merged**